dtfs-central-api/api-tests/v1/deals/create-deal.api-test.js

```diff
@@ -0,0 +1,135 @@
+const { MONGO_DB_COLLECTIONS } = require('@ukef/dtfs2-common');
```

Comment:
The tests in this folder have been split from the existing test files and not refactored.